utils/cliutils/cli_consts.go

```diff
@@ -13,6 +13,7 @@ const (
 	CmdCompletion     = "completion"
 	CmdPlugin         = "plugin"
 	CmdConfig         = "config"
+	CmdEnvironments   = "environments"
```

Comment:
"environments" doesn't seem the right title for this option. Since I can't think of a good description for "supported environment variables" I would just go for "options" - 

CmdOptions   = "options"